Resources/Prototypes/Floof/Loadouts/Jobs/Security/uncategorized.yml

```diff
@@ -1,4 +1,70 @@
 # Uncategorized
+
+- type: loadout
+  id: LoadoutMagazineSpecialRubber
+  category: JobsSecurityAUncategorized
+  cost: 2
+  exclusive: true
+  requirements:
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 3600 # 1 hours
+    - !type:CharacterItemGroupRequirement
```

Comment:
Note that these CharacterItemGroupRequirements won't work properly because you never added your items to the respective item groups. Not a big issue though, I personally think CIGs are stupid.